restli-common/src/main/java/com/linkedin/restli/common/validation/RestLiDataValidator.java

```diff
@@ -247,8 +250,7 @@ public RestLiDataValidator(Map<String, List<String>> annotations,
       for (Map.Entry<String, List<String>> entry : annotations.entrySet())
       {
         String annotationName = entry.getKey();
-        if (annotationName.equals(ReadOnly.class.getAnnotation(RestSpecAnnotation.class).name())
-            && readOnlyRestrictedMethods.contains(resourceMethod))
+        if (annotationName.equals(ReadOnly.class.getAnnotation(RestSpecAnnotation.class).name()))
```

Comment:
Should you also make the same change for the following if-statement (for createOnly) so that the way these predicates are constructed is consistent?

Reply:
> Should you also make the same change for the following if-statement (for createOnly) so that the way these predicates are constructed is consistent?

+1

Reply:
Done